Первая лабараторная

JIUMOH opened the pull request:
Студент Русаков Станислав.

JIUMOH committed:
Добавил 1,2,3 лабы

JIUMOH committed:
Добавил варианты

JIUMOH committed:
1

JIUMOH committed:
2

PashaKlybik reviewed (changes requested):
Удалить файл со второй лабораторной, что за задание, какой вариант, что надо было сделать в вашем варианте? 
НАЗВАНИЕ ПР ПОМЕНЯТЬ, ЧЬЯ ЛАБА?

JIUMOH committed:
Add files via upload

Лабараторная №1: Написать программу, которая помещает в регистр AX наибольшее из 4 чисел.
Лабараторная №2: Написать процедуру для вывода числа размерностью слово из регистра AX на экран
Лабараторная №3: Написать процедуру для вывода знакового числа размерностью слово из регистра AX на экран
Лабараторная №4: Посчитать, сколько слов начинается с гласной буквы
Лабараторная №5: Необходимо найти сумму элементов, находящихся в четных столбах и показать эту сумму на экране.
Лабараторная №6: Перекрыть девятую функцию прерывания 21h таким образом, чтобы в вводимой строке были продублированы все гласные буквы, а гласные игнорировались.
